### PR TITLE
Small doc improvements (annotations, implicite time in ODEs/SDEs)

### PR DESCRIPTION
**Pull request by pierreguilmin**
A bunch of minor improvements to the doc.

Before | after
Screenshot 2024-05-31 at 00 09 05

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ theme:
     repo: fontawesome/brands/github
     admonition:
       example: material/human-male-board
+      annotation: material/arrow-right-circle
```
Comment (gautierronan) [inline mkdocs.yml:22]: This doesn't seem to work, right? It's not an arrow which is used but a plus.

**Review by gautierronan — DISMISSED** [review]
I'm not really a fan of these annotations TBH. I tried to reduce their sizes in 0c9a7c35ebfb54d7587cef4b297c0af898f6f246 but I still don't find this 100% satisfactory.

I might be in favor of going back to footnotes, which makes it super clear that there's something additional to be read (here it's not super clear that the button is clickable).

I still approve the PR because it's not that important, and the other changes are good. Feel free to merge like that, or to revert back to footnotes if you agree with me.

File: mkdocs.yml (hunk 1 above)
Comment (pierreguilmin) [reply mkdocs.yml]: Right sorry, I just fixed it. See the doc here > https://squidfunk.github.io/mkdocs-material/reference/annotations/.

**Review by gautierronan — APPROVED** [review]

**Comment by pierreguilmin** [comment]
> I might be in favor of going back to footnotes, which makes it super clear that there's something additional to be read (here it's not super clear that the button is clickable).

Footnotes have two issues:
- they don't work in the API
- they jump the page completely to the end, it's really annoying when reading

I would say it's quite natural to be "what's that" and click. I think the current is much too small though, see the mkdocs-material doc for example > https://squidfunk.github.io/mkdocs-material/reference/annotations/.

**Comment by pierreguilmin** [comment]
I'll merge this and we can fix the style in another PR. Maybe we should use another icon? Like:
Screenshot 2024-06-06 at 22 02 13

**Pull request merged**